examples/tv-app/android/java/ContentLauncherManager.cpp

```diff
@@ -351,3 +155,124 @@ ContentLaunchResponse ContentLauncherManager::LaunchUrl(const chip::CharSpan & c
+            const char * convertedValue = (env)->GetStringUTFChars(jAcceptedHeader, JNI_FALSE);
+            std::string acceptedHeader  = std::string(convertedValue, strlen(convertedValue));
+            acceptedHeadersList.push_front(acceptedHeader);
+        }
+    }
+
+exit:
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(Zcl, "ContentLauncherManager::GetAcceptsHeader status error: %s", err.AsString());
+    }
+
+    return acceptedHeadersList;
+}
+
+uint32_t ContentLauncherManager::HandleGetSupportedStreamingProtocols()
+{
+    CHIP_ERROR err                       = CHIP_NO_ERROR;
+    JNIEnv * env                         = JniReferences::GetInstance().GetEnvForCurrentThread();
+    uint32_t supportedStreamingProtocols = 0;
+
+    ChipLogProgress(Zcl, "Received ContentLauncherManager::GetSupportedStreamingProtocols");
+    VerifyOrExit(mContentLauncherManagerObject != nullptr, err = CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrExit(mGetSupportedStreamingProtocolsMethod != nullptr, err = CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrExit(env != NULL, err = CHIP_JNI_ERROR_NO_ENV);
+
+    {
+        jint jSupportedStreamingProtocols =
+            env->CallIntMethod(mContentLauncherManagerObject, mGetSupportedStreamingProtocolsMethod);
+        supportedStreamingProtocols = (uint32_t) jSupportedStreamingProtocols;
+        if (env->ExceptionCheck())
+        {
+            ChipLogError(Zcl, "Java exception in ContentLauncherManager::GetAcceptsHeader");
+            env->ExceptionDescribe();
+            env->ExceptionClear();
+            err = CHIP_ERROR_INCORRECT_STATE;
+            goto exit;
+        }
+    }
+
+exit:
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(Zcl, "ContentLauncherManager::GetSupportedStreamingProtocols status error: %s", err.AsString());
+    }
+
+    return supportedStreamingProtocols;
+}
+
+void ContentLauncherManager::InitializeWithObjects(jobject managerObject)
+{
+    JNIEnv * env = JniReferences::GetInstance().GetEnvForCurrentThread();
+    VerifyOrReturn(env != nullptr, ChipLogError(Zcl, "Failed to GetEnvForCurrentThread for ContentLauncherManager"));
+
+    mContentLauncherManagerObject = env->NewGlobalRef(managerObject);
+    VerifyOrReturn(mContentLauncherManagerObject != nullptr, ChipLogError(Zcl, "Failed to NewGlobalRef ContentLauncherManager"));
+
+    jclass ContentLauncherClass = env->GetObjectClass(managerObject);
+    VerifyOrReturn(ContentLauncherClass != nullptr, ChipLogError(Zcl, "Failed to get ContentLauncherManager Java class"));
+
+    mGetAcceptsHeaderMethod = env->GetMethodID(ContentLauncherClass, "getAcceptsHeader", "()[Ljava/lang/String;");
+    if (mGetAcceptsHeaderMethod == nullptr)
+    {
+        ChipLogError(Zcl, "Failed to access MediaInputManager 'getInputList' method");
+        env->ExceptionClear();
+    }
+
+    mGetSupportedStreamingProtocolsMethod = env->GetMethodID(ContentLauncherClass, "getSupportedStreamingProtocols", "()[I");
```

Comment:
you changed the function name of jni, so did you also changed the java file and test them?

Reply:
No. I've updated it now.